Azure/sonic-buildimage
# [device/Accton] Fix i2c fault from ir3570a on 3 models

I2c burst read may failed due to misoperation of ir3570a(A DC-to-DC converter IC).
As https://github.com/Azure/sonic-buildimage/pull/2966, there are 3 more models have this symptom, as7326-56x, as7726-32x, and as9716-32d.
Also correct typo of naming on as7816-64x and as7716-32x.


Signed-off-by: roy_lee roy_lee@accton.com

- What I did
Disabling i2c function of ir3570a which may failed i2c tranfer to others.
Close channel of mux after data transfered.
- How I did it

Identify version of ir3570, if it's ir3570a, disable its alias i2c address.
Enable parameter of driver i2c_mux_pca954x to close channel on after every access.
- How to verify it

Write 08 to offset 0xcf of systom eeprom and execute i2c block read.
It will return error.
plug-in several transceivers and run "show interfaces transceiver presence" and "sfputil show presence". You may see the different result. (But doing this requires updated xcvrd).
- Description for the changelog
As the code diff.
